### ISSUE TITLE
OWL Day 1 - Could not get player data, keeping stream status as Offline.

### ISSUE THREAD

**Issue by Kronman590** [ucarno/ow-league-tokens#43]
Still getting this error message, I don't believe my Id is incorrect either. The "Connected" button is on the Youtube player so I believe tokens should be farmable.

**Comment by ucarno** [comment]
Stream is not available on OWL website, I will be rewriting this bot so it works on YouTube

**Comment by Kronman590** [comment]
Thank you for your efforts 🙏

**Comment by Aezriel82** [comment]
> Stream is not available on OWL website, I will be rewriting this bot so it works on YouTube

any idea if u will have something ready for day 2 (17hours away) to test? again thanks for efforts and no pressure.  

FYI it seems that shirokumacode(omnic rewards guy) is no longer developing his farmer since DEC22 and it also appears to be broken now too. likely the same issue??

**Comment by ucarno** [comment]
> any idea if u will have something ready for day 2 (17hours away) to test? again thanks for efforts and no pressure.  

Yes, but I am not sure about YouTube's bot detection and if watching headlessly (no bot's browser window, just console as before) will give tokens. So I will need feedback on this.

**Comment by MrHugelberg** [comment]
> > any idea if u will have something ready for day 2 (17hours away) to test? again thanks for efforts and no pressure.
> 
> Yes, but I am not sure about YouTube's bot detection and if watching headlessly (no bot's browser window, just console as before) will give tokens. So I will need feedback on this.

Would love to give feedback. Yesterday I got token immediately so this time it shouldn't take as long to check if it works.

**Comment by LowHighh** [comment]
thoughts on making a discord server?

**Comment by christianb4** [comment]
can't wait to have a token farmer that works, OWL stream keeps crashing for me

**Comment by ucarno** [comment]
Made an update. Executable app later.

**Comment by Coathar** [comment]
Seems like it's working, logged in and then swapped to headless mode and I've been getting tokens from the broadcast

**Comment by Aezriel82** [comment]
how did it go? i couldnt test myself. was it  working well enuf to get ur youtube update into the exe?

**Comment by CrestfallenWarrior** [comment]
I can't test myself either. OWL is live right now and no diamond is showing up so I'm not getting tokens by watching either

**Comment by ucarno** [comment]
Issue related to legacy version
